File: C/Beecrowd/bee-3454.c
#include <stdio.h>

int main(){
    char game[3];
    int O = 0;
    for(int i = 0; i < 3; i++){
        scanf("%c", &game[i]);
        if(game[i] == 'O'){ O++; }
    }
    if(O != 1){ printf("?\n"); }
    else if(game[1] == 'O'){
        printf("*\n");
    }
    else{ printf("Alice\n"); }

    return 0;
}

/* 3454 - Alice in Noughts and Crosses Wonderland
https://www.beecrowd.com.br/judge/en/problems/view/3454 */

File: Python/Beecrowd/bee-3454.py
game = input()
O = 0
for i in range(0, 3):
    if game[i] == 'O':
        O += 1

if O != 1:
    print('?')
elif game[1] == 'O':
    print('*')
else:
    print('Alice')

# 3454 - Alice in Noughts and Crosses Wonderland
# https://www.beecrowd.com.br/judge/en/problems/view/3454
